src/pages/index.tsx

```diff
@@ -1,7 +1,4 @@
 import Head from "next/head";
-import { PrismaClient } from "@prisma/client";
-
-export const prisma = new PrismaClient();
```

Comment:
if you remove this, you need to redirect all the paths of files that import this export to the correct prisma export in `pages/api/prisma.ts`. 
See autocomplete.ts, addItem.ts, and the seed.ts file which also instantiates a prisma client.